src/osd/PG.cc

```diff
@@ -1428,15 +1428,28 @@ bool PG::choose_acting(pg_shard_t &auth_log_shard_id,
     }
   }
 
-  // We go incomplete if below min_size for ec_pools since backfill
+  // We go incomplete if below data_chunk count for ec_pools since backfill
   // does not currently maintain rollbackability
```

Comment:
Oh, and I think now we can just drop this comment. We'll only go incomplete if we are, in fact, incomplete. Which is the expected behavior.